interop-testing/src/main/java/io/grpc/testing/integration/XdsTestServer.java

```diff
@@ -180,7 +180,7 @@ private void start() throws Exception {
           Grpc.newServerBuilderForPort(maintenancePort, InsecureServerCredentials.create())
               .addService(new XdsUpdateHealthServiceImpl(health))
               .addService(health.getHealthService())
-              .addService(ProtoReflectionService.newInstance())
+              .addService(ProtoReflectionServiceV1.newInstance())
```

Comment:
Keep the old service here and later in this file, as well. (Have both old+new)

Reply:
Done.

Reply:
File got missed in earlier commit, done now.

Reply:
Below (line 205), was still missed.

Reply:
Oops, done now.